libs/ui/src/big_metric.tsx

```diff
@@ -24,9 +24,7 @@ const StyledContainer = styled.span`
 
 const StyledValue = styled(H1)`
   display: inline-block;
-
-  /* TODO: Remove once VxScan is updated to default to the new VVSG size theme: */
-  font-size: ${(p) => (p.theme.sizeMode === 'legacy' ? '72px' : undefined)};
+  font-size: 72px;
```

Comment:
I think this needs to be removed for parity with the old behaviour

Reply:
Oh whoops, that was a mistake. Thanks for catching!